apps/frontend/src/components/chat/ChatBubble.tsx

```
@@ -1,102 +1,105 @@
+					{message.citations && message.citations.length > 0 && (
+						<div className="mt-3 space-y-2">
+							{message.citations.map((citation) => (
+								<div key={citation.id} className="text-sm">
+									<button
+										className={`flex items-center ${
+											message.sender === "user"
+												? "text-blue-200"
+												: "text-blue-600"
+										}`}
+										onClick={() => toggleCitation(citation.id)}
+									>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)